setup.py

```diff
@@ -56,7 +56,6 @@ def get_version():
         "stripe",
         "authlib",
     ],
-    extras_require={"java": ["pyjnius"]},
```

Comment:
Not entirely sure about removing this, but since it was added in #1990 I'm defaulting to taking it out.